packages/rn-tester/js/examples/Alert/AlertExample.js

```diff
@@ -9,119 +9,188 @@
-          onPress={() =>
-            Alert.alert(
-              'Alert Title',
-              null,
-              [{text: 'OK', onPress: () => console.log('OK Pressed!')}],
-              {
-                cancelable: false,
-              },
-            )
-          }>
-          <View style={styles.button}>
-            <Text>Alert that cannot be dismissed</Text>
-          </View>
-        </TouchableHighlight>
-        <TouchableHighlight
-          style={styles.wrapper}
-          onPress={() =>
-            Alert.alert('', alertMessage, [
-              {text: 'OK', onPress: () => console.log('OK Pressed!')},
-            ])
-          }>
-          <View style={styles.button}>
-            <Text>Alert without title</Text>
-          </View>
-        </TouchableHighlight>
-      </View>
-    );
-  }
-}
+
+const AlertWithDefaultButton = () => {
+  const alertMessage = 'An external USB drive has been detected!';
+
+  return (
+    <View>
+      <TouchableHighlight
+        testID="alert-with-default-button"
+        style={styles.wrapper}
+        onPress={() => Alert.alert('Alert', alertMessage)}>
+        <View style={styles.button}>
+          <Text>Tap to view alert</Text>
+        </View>
+      </TouchableHighlight>
+    </View>
+  );
+};
+
+const AlertWithTwoButtons = () => {
+  const [message, setMessage] = useState('');
+
+  const alertMessage = 'Your subscription has expired!';
+
+  return (
+    <View>
+      <TouchableHighlight
+        style={styles.wrapper}
+        onPress={() =>
+          Alert.alert('Action Required!', alertMessage, [
+            {text: 'Ignore', onPress: () => setMessage('Ignore Pressed!')},
+            {text: 'Renew', onPress: () => setMessage('Renew Pressed!')},
+          ])
+        }>
+        <View style={styles.button}>
+          <Text>Tap to view alert</Text>
+        </View>
+      </TouchableHighlight>
+      <Log message={message} />
+    </View>
+  );
+};
+
+const AlertWithThreeButtons = () => {
+  const [message, setMessage] = useState('');
+
+  const alertMessage = 'Do you want to save your changes?';
+
+  return (
+    <View>
+      <TouchableHighlight
+        testID="alert-with-three-buttons"
+        style={styles.wrapper}
+        onPress={() =>
+          Alert.alert('Unsaved Changes!', alertMessage, [
+            {text: 'Cancel', onPress: () => setMessage('Cancel Pressed!')},
+            {text: 'No', onPress: () => setMessage('No Pressed!')},
+            {text: 'Yes', onPress: () => setMessage('Yes Pressed!')},
+          ])
+        }>
+        <View style={styles.button}>
+          <Text>Tap to view alert</Text>
+        </View>
+      </TouchableHighlight>
+      <Log message={message} />
+    </View>
+  );
+};
+
+const AlertWithManyButtons = () => {
+  const [message, setMessage] = useState('');
+
+  const alertMessage =
+    'Credibly reintermediate next-generation potentialities after goal-oriented ' +
+    'catalysts for change. Dynamically revolutionize.';
```

Comment:
hahaha what?

Reply:
haha.. It corporate ipsum. It was already there in the file. I just used it for `AlertWithManyButtons` example because I couldn't come up with any alert message which would require more than 3 buttons.